scripts/create_summary.ts

```diff
@@ -0,0 +1,94 @@
+import {Changes, ConfigurationOptions} from '../src/schemas'
```

Comment:
This addition raises an error when running `tsc`:

```
~/w/dr-action (adjust_summary_format)
$ tsc -p tsconfig.build.json
error TS6059: File '/Users/febuiles/w/dr-action/scripts/create_summary.ts' is not under 'rootDir' '/Users/febuiles/w/dr-action/src'. 'rootDir' is expected to contain all source files.
  The file is in the program because:
    Matched by default include pattern '**/*'
```

I originally thought of adding `./scripts` to `rootDirs`, but I don't think it's great we start vendoring the JS/TS files in the repo for a script I think won't be used often. Should we exclude this from the eslint build config, or what do you think?

Reply:
Whoops. Missed that one, thanks for spotting!

I adjusted the `tsconfig.build.json` now to only include the source library. This should fix the error and also prevent any other resources (the fixtures and the script) from being built or included in the bundle.

I am totally with you in not compiling and/or building the scripts-folder. I also added some explanation now in how to use it.